frontend/containers/DocumentList/DocumentList.jsx

```diff
@@ -75,7 +75,7 @@ class DocumentList extends Component {
   constructor(props) {
     super(props)
 
-    this.state.selectedRows = {}
+    this.state = this.defaultLocalState()
```

Comment:
State within components should always remain thin and lightweight, as the heavy-lifting should always belong in the store. If we find ourselves with pretty large state at the component level, we're probably doing something wrong.